src/components/Modal/README.md

```diff
@@ -658,7 +658,7 @@ function ModalExample() {
 
   const handleClose = useCallback(() => {
     setActive(false);
-    button.current.querySelector('button').focus();
+    requestAnimationFrame(() => button.current.querySelector('button').focus());
```

Comment:
I wonder if it's worth making this work without RAF? It's not very intuitive using it to focus an element and I'm if we're supporting this we'll get questions about it in the future.

Reply:
I agree that it's not ideal, but since we're going to make the `activator` prop mandatory in the future (give or take a few weeks after the release of v5), this example will be entirely removed. For that short while, think it's okay to leave it as is?